pkg/controller/backup_sync_controller.go

```diff
@@ -162,44 +134,44 @@ func (c *backupSyncController) run() {
 
 	for _, location := range locations {
 		log := c.logger.WithField("backupLocation", location.Name)
+		log.Info("Attempting to sync contents of backup storage location into cluster")
 
 		backupStore, err := c.newBackupStore(location, pluginManager, log)
 		if err != nil {
 			log.WithError(err).Error("Error getting backup store for this location")
 			continue
 		}
 
-		ok, revision := shouldSync(location, time.Now().UTC(), backupStore, log)
-		if !ok {
-			continue
-		}
-		log.Info("Syncing contents of backup store into cluster")
-
+		// get a list of all the backups that are stored in the backup storage location
 		res, err := backupStore.ListBackups()
 		if err != nil {
 			log.WithError(err).Error("Error listing backups in backup store")
 			continue
 		}
 		backupStoreBackups := sets.NewString(res...)
-		log.WithField("backupCount", len(backupStoreBackups)).Info("Got backups from backup store")
+		log.WithField("backupCount", len(backupStoreBackups)).Debug("Got backups from backup store")
 
-		for backupName := range backupStoreBackups {
-			log = log.WithField("backup", backupName)
-			log.Debug("Checking this backup to see if it needs to be synced into the cluster")
+		// get a list of all the backups that exist as custom resources in the cluster
+		clusterBackups, err := c.backupLister.Backups(c.namespace).List(labels.Everything())
```

Comment:
curious: what's the difference between `labels.Everything()` and an empty `metav1.ListOptions{}`?

Reply:
this is using a Lister, which doesn't take a `ListOptions` like the clients do -- it only takes a label selector. So same effect, but different types.